applications/GeoMechanicsApplication/tests/cpp_tests/test_transient_thermal_element.cpp

```diff
@@ -0,0 +1,428 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "containers/model.h"
+#include "custom_elements/transient_thermal_element.h"
+#include "geo_mechanics_application_variables.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing {
+
+void TestThermalElement(ModelPart& rModelPart)
+{
+    Element::Pointer p_element = rModelPart.pGetElement(1);
+
+    for (unsigned int i = 0; i < rModelPart.NumberOfNodes(); i++) {
+        p_element->GetGeometry()[i].AddDof(TEMPERATURE);
+    }
+
+    const ProcessInfo& r_current_process_info = rModelPart.GetProcessInfo();
+    // p_element->Check(r_current_process_info);
```

Comment:
This is still commented, what do we want to do with this?

Reply:
I have removed it.